### ISSUE TITLE
Demo is not working.

### ISSUE THREAD

**Issue by rajeshwar-fissionlabs** [klaemo/popover#6]
Demo is not working.

**Comment by ghost** [comment]
Yup , Uncaught TypeError: Failed to execute 'removeEventListener' on 'EventTarget': 2 arguments required, but only 1 present.

**Comment by Riloko** [comment]
Hi, you have to change this - popover.el.removeEventListener('click')
To this - popover.off()

can't contribute to this project(
